apparmor.d/profiles-a-f/atool

```diff
@@ -0,0 +1,52 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2019-2021 Mikhail Morfikov
+# Copyright (C) 2023-2024 Alexandre Pujol <alexandre@pujol.io>
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{exec_path} = @{bin}/atool
+profile atool /{,usr/}{,s}bin/atool {
+  include <abstractions/base>
+  include <abstractions/user-write>
```

Comment:
Use both `<abstractions/user-write-strict>` and `<abstractions/user-read-strict>` instead. Same elsewhere.

Reply:
To be clear, this prevents user from extracting archives located in @{HOME}/ or reading any metadata from files directly located there.
If this is intended, ok. But I do not see what this adds in terms of protection while it will likely get in the way of the user.

Reply:
As you may remember the security boundaries we try to maintain: https://github.com/roddhjav/apparmor.d/discussions/307#discussioncomment-8944433. Unlabelled folder/file would be a source of problem (as in: we lost control over what is acceded), then we try to ensure that every user data folder/file is labelled in such a way it can be controlled, revoked, and we can prevent/decrease leak between profile.